packages/core/src/modules/ledger/LedgerModule.ts

```diff
@@ -72,4 +72,12 @@ export class LedgerModule {
   public async getCredentialDefinition(id: string) {
     return this.ledgerService.getCredentialDefinition(id)
   }
+
+  public async getRevocationRegistryDefinition(revocRegDefId: string) {
```

Comment:
Nit: We use full names everywhere in AFJ, especially to avoid inconsistency in naming (e.g. this method uses `revocRegDefId` and the method below uses `revRegId`.

Maybe we can use the full `revocationRegistryDefinitionId` here? Is there a difference between `revocationRegistryId` and `revocationRegistryDefinitionId`? If not maybe we can use `revocationRegistryId` everywere?

Reply:
And there's no difference between the two ids--they're identical 👍